examples/ios/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: xcodebuild run install boot-sim generate clean
+
+run: install
+	xcrun simctl launch --console $(DEVICE_ID) com.rust.bevy-ios-example
+
+boot-sim:
+	xcrun simctl boot $(DEVICE_ID) || true
+
+install: xcodebuild boot-sim
+	 xcrun simctl install $(DEVICE_ID) build/Build/Products/Debug-iphonesimulator/bevy_ios_example.app
+
+DEVICE_ID=$(shell xcrun simctl list  devices 'iOS' | grep -v '^--' | grep -v '==' | head -n 1 | cut -d ' ' -f 7 | sed 's/[()]//g')
```

Comment:
can we make this settable from the shell env? that way I can override with a specific device ids, i.e. my phone and my preferred simulator

Reply:
Done.